<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachingDialog</class>
 <widget class="QDialog" name="TeachingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Введите параметры обучения</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>25</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="lbl_inputs">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Количество входов:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbl_outputs">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Количество выходов:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbl_hidLayCount">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Количество скрытых слоев:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbl_eraCount">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Количество эпох:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbl_nu">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Скорость обучения:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbl_minMistake">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Минимальная ошибка:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>25</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <item>
               <widget class="QLineEdit" name="lnEditInputs">
                <property name="text">
                 <string>1024</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditOutputs">
                <property name="text">
                 <string>43</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditHiddenLayCount">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditEraCount">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditNu">
                <property name="text">
                 <string>0.05</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditMinMistake">
                <property name="text">
                 <string>0.001</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Активационные функции:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="chBox_Tanh">
              <property name="text">
               <string>TanH</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chBox_Logsigm">
              <property name="text">
               <string>Log-sigmoid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chBox_Softplus">
              <property name="text">
               <string>Softplus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chBox_ReLU">
              <property name="text">
               <string>ReLU</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chBox_RReLU">
              <property name="text">
               <string>Randomized ReLU</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Нейронов в скрытых слоях:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTextEdit" name="txtEditHiddenNeurons">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_teach">
         <property name="text">
          <string>Обучить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
